src/Components/VisualsModal/VisualsModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+    IBehavior,
+    ITwinToObjectMapping,
+    IVisual
+} from '../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
+import ElementsList from '../ElementsPanel/Internal/ElementsList';
+import { IViewerElementsPanelItem } from '../ElementsPanel/ViewerElementsPanel.types';
+import { getStyles } from './VisualsModal.styles';
+
+export interface IVisualsModalProps {
+    badges: IViewerElementsPanelItem;
+    onClose: () => any;
+    position: { left: number; top: number };
+    onItemClick: (
+        item: ITwinToObjectMapping | IVisual,
+        panelItem: IViewerElementsPanelItem,
+        behavior?: IBehavior
+    ) => void;
+    onItemHover?: (
+        item: ITwinToObjectMapping | IVisual,
+        panelItem: IViewerElementsPanelItem,
+        behavior?: IBehavior
+    ) => void;
+    onItemBlur?: (
+        item: ITwinToObjectMapping | IVisual,
+        panelItem: IViewerElementsPanelItem,
+        behavior?: IBehavior
+    ) => void;
+}
+
+const VisualsModal: React.FC<IVisualsModalProps> = ({
+    onClose,
+    position,
+    badges,
+    onItemClick,
+    onItemBlur,
+    onItemHover
+}) => {
+    const styles = getStyles();
+
+    return (
+        <div
+            style={{ top: position.top, left: position.left }}
+            className={styles.boundaryLayer + ' cb-base-fade-in'}
```

Comment:
nit: there's something about doing class names this way that sometimes causes issues. Can you use the `css` helper that `fluent-ui` exposes to combine these class names?